tests/test_for_support/test_for_basic.py

```diff
@@ -480,6 +480,44 @@ def dup_rxns_multiple_list_anns_no_match(base):
     return base
 
 
+@register_with(MODEL_REGISTRY)
+def rxns_no_exchange(base):
+    """Provide a model with no exchange reactions"""
+    met_a = cobra.Metabolite("a_c", compartment="c")
+    met_b = cobra.Metabolite("b_c", compartment="c")
+    rxn_1 = cobra.Reaction("rxn1")
+    rxn_1.add_metabolites({met_a: -1, met_b: 1})
+    return base
+
+
+@register_with(MODEL_REGISTRY)
+def rxns_with_two_substrates(base):
+    """Provide a model with two substrates that can be taken up"""
+    met_a = cobra.Metabolite("a_c", compartment="c")
+    met_b = cobra.Metabolite("b_c", compartment="c")
+    met_c = cobra.Metabolite("c_e", compartment="e")
+    met_d = cobra.Metabolite("d_e", compartment="e")
+    rxn_1 = cobra.Reaction("rxn1")
+    rxn_1.add_metabolites({met_a: -1, met_b: 1})
+    base.add_boundary(met_c, reaction_id="EX_c_e", lb=-1000, ub=1000)
```

Comment:
The parameter `reaction_id` is only used for custom reactions (i.e. `type="custom"`), hence it's superfluous here as the reaction name will default to `Ex_c_e`.

This is also true for the parameter `lb`. It is overridden by the `ub` parameter unless the `type` is set to custom.